docs/user/ppl/admin/jdbc.rst

```diff
@@ -0,0 +1,76 @@
+.. highlight:: sh
+
+====================
+Prometheus Connector
+====================
+
+.. rubric:: Table of contents
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+Introduction
+============
+
+This page covers JDBC connector properties for dataSource configuration and the nuances associated with JDBC connector.
+
+
+JDBC Connector Properties in DataSource Configuration
+=====================================================
+JDBC Connector Properties.
+
+* ``url`` [Required].
+    * This parameters provides the URL to connect to a database instance provided endpoint.
+* ``driver`` [Required].
+    * This parameters provides the Driver to connect to a database instance provided endpoint. The only supported ``org.apache.hive.jdbc.HiveDriver``
```

Comment:
Why only `HiveDriver` supported? There is no hardcoded restriction. Can I use anything else, for example, put `SQLite` driver into `CLASSPATH` and use it?

Reply:
Why only HiveDriver supported? There is no hardcoded restriction. 
> Yes, only hive supported. Explain the possible value in doc.

>  Can I use anything else, for example, put SQLite driver into CLASSPATH and use it?
You can, if you put SQLLite into classpath. But it is not prebuild.